internal/impl/shell.go

```diff
@@ -354,11 +343,6 @@ func (s *DevboxShell) writeDevboxShellrc() (path string, err error) {
 		}
 	}()
 
-	pathPrepend := s.profileDir + "/bin"
-	if s.pkgConfigDir != "" {
-		pathPrepend = s.pkgConfigDir + ":" + pathPrepend
-	}
```

Comment:
I understand that the profileDir should no longer be added to PATH
but why remove the plugins' pkgConfigDir?

Reply:
`pathPrepend` is only used when `ExportEnv` is empty. But `ExportEnv` is never empty ever since unified env is on.